modules/memo/src/main/Picfit.scala

```diff
@@ -22,7 +22,15 @@ case class PicfitImage(
     name: String,
     size: Int, // in bytes
     createdAt: Instant,
-    context: Option[String]
+    context: Option[String],
+    automod: Option[ImageAutomod] = none
+    // need a urls array here that tracks generated urls (including sigs) for cf purge
+    // operations on delete. there is no regex or partial query string wildcard support
```

Comment:
do we need to purge them from CF? Isn't it enough that we no longer link them from lichess.org?